scout/server/blueprints/cases/controllers.py

```diff
@@ -333,6 +333,9 @@ def case(store, institute_obj, case_obj):
     # Limit secondary findings according to institute settings
     limit_genes = store.safe_genes_filter(institute_obj["_id"])
 
+    # Make sure build is either "37" or "38"
+    build = "38" if "38" in str(case_obj["genome_build"]) else "37"
```

Comment:
```suggestion
    build = "38" if "38" in str(case_obj["genome_build"]) else "37"
```
```suggestion
    build = "38" if "38" in str(case_obj.get("genome_build")) else "37"
```

Reply:
I mean, it really should exist, but sure.

Reply:
It doesn't for some very old cases. I remember this caused a bug a few years ago